components/MarkdownTextBlock/samples/MarkdownTextBlockLiveEditorSample.xaml

```diff
@@ -11,23 +11,22 @@
     <Grid>
         <Grid.RowDefinitions>
             <RowDefinition Height="auto" />
-            <RowDefinition Height="auto" />
-            <RowDefinition Height="auto" />
-            <RowDefinition Height="auto" />
+            <RowDefinition Height="*" />
         </Grid.RowDefinitions>
         <TextBlock Grid.Row="0"
                    Margin="0,0,0,12"
                    FontSize="16"
                    FontWeight="Bold"
-                   Text="Try it live!" />
+                   Text="Try it live! Type in the text box below and see the markdown rendered in real-time:" />
         <Grid Grid.Row="1">
```

Comment:
Should we collapse this grid down? Would it just be easier to have the textbox on top and then the live preview on the bottom to better make use of the vertical space? Or maybe put the textbox on the left and the markdown preview on the right more like a markdown editor?